api/src/main/java/io/opencensus/common/Duration.java

```diff
@@ -78,6 +79,20 @@ public static Duration fromMillis(long millis) {
     return Duration.create(seconds, nanos);
   }
 
+  /**
+   * Converts a {@link Duration} to milliseconds.
+   *
+   * <p>Note that there could be overflow or loss of precision by making this conversion. See {@link
+   * TimeUnit#convert(long, TimeUnit)} for details.
+   *
+   * @param duration a {@code Duration}.
+   * @return the milliseconds representation of this {@code Duration}.
+   */
+  public static long toMillis(Duration duration) {
```

Comment:
I would rather make this method non-static, for consistency with getSeconds and getNanos.

Reply:
Done.